### ISSUE TITLE
SpacingAroundCurly misfiring when NoEmptyClassBody autoCorrect it on

### ISSUE THREAD

**Issue by hick209** [detekt/detekt#1634]
## Expected Behavior
`SpacingAroundCurly` should only fire if there are missing spaces around curly braces.

## Observed Behavior
Consistent false positive for `SpacingAroundCurly` when `NoEmptyClassBody.autoCorrect = true` and there's an issue of type `NoEmptyClassBody`.

## Steps to Reproduce
Configuration:
```
autoCorrect: false

formatting:
  active: true
  autoCorrect: true
  NoEmptyClassBody:
    active: true
    autoCorrect: true
  SpacingAroundCurly:
    active: true
```

File:
```
class EmptyClass {}
```

Findinds:
```
Ruleset: formatting - 10min debt
	NoEmptyClassBody - [Element(CLASS_BODY)] at EmptyClass.kt:1:18
	SpacingAroundCurly - [PsiElement(LBRACE)] at EmptyClass.kt:1:1
```

To the issue will no longer happen if you set the `autoCorrect` from `NoEmptyClassBody` to `false`.

## Your Environment
* Version of detekt used: 1.0.0.RC14 (tested on master as well)
* Version of Gradle used (if applicable): N/A
* Operating System and version: macOS
* Link to your project (if it's a public repository): N/A

**Comment by 3flex** [comment]
Can I ask if you can replicate using ktlint directly?

**Comment by hick209** [comment]
I tried, but I could not replicate it in KtLint

**Comment by arturbosch** [comment]
I could reproduce this issue.
KtLint uses two runs, one for formatting and an additional for reporting the remaining errors.
We just use one and have a config flag for this issue:  `excludeCorrectable: true`.

Please update your config to this:
```yaml
build:
  maxIssues: 0
  excludeCorrectable: true

formatting:
  active: true
  autoCorrect: true
  NoEmptyClassBody:
    active: true
    autoCorrect: true
  SpacingAroundCurly:
    active: true
```
